src/main/js/main_html5.js

```diff
@@ -336,8 +336,13 @@ require("../../../html5-common/js/utils/environment.js");
           _video.pause();
         } catch (ex) {
           // error because currentTime does not exist because stream hasn't been retrieved yet
-          console.log('VTC_OO: Failed to rewind video, probably ok; continuing');
+          OO.log('VTC_OO: Failed to rewind video, probably ok; continuing');
         }
+      } else if (!!rewind && OO.isEdge) {
+        // PBW-4555: Edge browser will always go back to time 0 on load.  Setting time to 0 here would
+        // cause the raw video element to enter seeking state.  If we call load while seeking on Edge,
+        // then seeking no longer works until the video stream url is changed.
+        currentTime = 0;
```

Comment:
I had that at first and tried a few other ways but ended up changing it because the logic for each was totally separate.  That would have been useful if they some same logic.  So the above just nests it deeper into a common if.  I could do it that way... hmm, not sure since i changed it away from that on purpose